package.json

```diff
@@ -148,10 +149,11 @@
     "enzyme-to-json": "^3.4.0",
     "eslint": "^6.4.0",
     "eslint-config-airbnb": "^18.0.1",
-    "eslint-import-resolver-alias": "^1.1.2",
     "eslint-config-prettier": "^6.3.0",
+    "eslint-import-resolver-alias": "^1.1.2",
     "eslint-plugin-cypress": "^2.6.1",
     "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-jam3": "^0.2.1",
```

Comment:
what about creating our own custom eslint plugin with our own custom rules instead of using another company's plugin with their own rules? I don't think this is too difficult to pull off and could be something that we iterate on and add add more custom rules to as we see fit.

https://eslint.org/docs/developer-guide/working-with-rules
https://eslint.org/docs/developer-guide/working-with-plugins